java-core/src/main/java/ai/certifai/junit5/ex1/BasicMethodAssertion.java

```diff
@@ -1,12 +1,12 @@
-package solution;
+package ai.certifai.junit5.ex1;
```

Comment:
the folder naming i suggest to change it to "unittest" instead of "junit5". unittest is more topical based and junit5 is the name of the library